### PR TITLE
test(docker): use named volumes

### PR DESCRIPTION
**Pull request by simitt**
cc @beniwohli 

Change to use a named volume as we experienced repeated docker issues with stopping containers.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -21,11 +23,15 @@ services:
...
     ports: 
       - "3306:3306"
+    volumes:
+      - nodemongodata:/data/db
 
   redis:
     image: redis
     ports:
       - "6379:6379"
+    volumes:
```
Comment (beniwohli) [inline docker-compose.yml]: I don't think we need this for redis, IIRC persistence is disabled by default in the docker redis image

**Comment by simitt** [comment]
sorry for the noise - I updated the other volumes accordingly now.

**Comment by simitt** [comment]
jenkins, retest this please.

**Comment by watson** [comment]
jenkins, run the tav tests

**Comment by simitt** [comment]
jenkins, run the tav tests

**Pull request merged**